### ISSUE TITLE
Hot module generator fails if there is no application.html.erb

### ISSUE THREAD

**Issue by dbackeus** [netguru/react_webpack_rails#81]
```
/Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/react_webpack_rails-0.3.1/lib/generators/react_webpack_rails/install/hot_reload_generator.rb:36:in `extname': no implicit conversion of nil into String (TypeError)
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/react_webpack_rails-0.3.1/lib/generators/react_webpack_rails/install/hot_reload_generator.rb:36:in `template_language_settings'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/react_webpack_rails-0.3.1/lib/generators/react_webpack_rails/install/hot_reload_generator.rb:26:in `partial'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `block in invoke_all'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `each'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `map'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `invoke_all'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/group.rb:232:in `dispatch'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.2/lib/rails/generators.rb:157:in `invoke'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.2/lib/rails/commands/generate.rb:13:in `<top (required)>'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/dependencies.rb:274:in `require'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/dependencies.rb:274:in `block in require'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/dependencies.rb:240:in `load_dependency'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/activesupport-4.2.5.2/lib/active_support/dependencies.rb:274:in `require'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.2/lib/rails/commands/commands_tasks.rb:123:in `require_command!'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.2/lib/rails/commands/commands_tasks.rb:130:in `generate_or_destroy'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.2/lib/rails/commands/commands_tasks.rb:50:in `generate'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.2/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
	from /Users/davidbackeus/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/railties-4.2.5.2/lib/rails/commands.rb:17:in `<top (required)>'
```

**Comment by mxaly** [comment]
@dbackeus fix is on the way. To make it work anyway, just add to you main layout file `<%= render 'layouts/react_hot_assets'%>

**Comment by mxaly** [comment]
fixed in #82
